cam.go

```diff
@@ -344,10 +348,10 @@ func (fc *filteredCamera) Image(ctx context.Context, mimeType string, extra map[
 		return nil, camera.ImageMetadata{}, err
 	}
 
-	return ImagesToImage(ctx, ni, mimeType)
+	return ImagesToImage(ctx, ni)
 }
 
-func (fc *filteredCamera) Images(ctx context.Context, extra map[string]interface{}) ([]camera.NamedImage, resource.ResponseMetadata, error) {
+func (fc *filteredCamera) Images(ctx context.Context, filterSourceNames []string, extra map[string]interface{}) ([]camera.NamedImage, resource.ResponseMetadata, error) {
```

Comment:
We're ignoring the `filterSourceNames` arg passed in here. Are there any cases where we don't want to process certain source names and would want `filterSourceNames` handled?